src/content/docs/products/splashkit/SplashKit Tutorials/Documentation/4-adding-oop.mdx

```diff
@@ -0,0 +1,188 @@
+---
+title: Guide to adding OOP to SplashKit tutorials
+sidebar:
+  label: 4. Adding OOP to Guides
+  order: 4
+---
+
+import { Tabs, TabItem } from "@astrojs/starlight/components";
+
+## Adding OOP and Top Level C# to Splashkit Tutorials
+
+One of the current goals of the SplashKit team is to enhance the tutorials by including both top-level statement and object-oriented (OOP) versions of C# programs. This guide outlines how to effectively integrate OOP into the existing tutorials, providing both options for users to choose from.
+
+### The Full Code Block Structure
+
+The full code block structure for C++, C# in top level and OOP, and Python is as follows:
+
+````md
+<Tabs>
```

Comment:
This Tabs group is just missing the syncKey of "code-language" like you have at the bottom. Everything else looks good.

Reply:
fixed :)